apm-sniffer/optional-plugins/play-2.x-plugin/pom.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one or more
+  ~ contributor license agreements.  See the NOTICE file distributed with
+  ~ this work for additional information regarding copyright ownership.
+  ~ The ASF licenses this file to You under the Apache License, Version 2.0
+  ~ (the "License"); you may not use this file except in compliance with
+  ~ the License.  You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.apache.skywalking</groupId>
+        <artifactId>apm-sdk-plugin</artifactId>
+        <version>6.4.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>apm-play-2.x-plugin</artifactId>
+    <name>play-2.x-plugin</name>
+    <packaging>jar</packaging>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <play2.version>2.7.3</play2.version>
+        <java.version>1.8</java.version>
+        <maven.compiler.source>${java.version}</maven.compiler.source>
+        <maven.compiler.target>${java.version}</maven.compiler.target>
+    </properties>
```

Comment:
I remember that `<compiler.version>1.8</compiler.version>` is enough to compile with 1.8
Can you try and test, this may be a simpler `pom.xml` file..

Reply:
https://maven.apache.org/plugins/maven-compiler-plugin/examples/set-compiler-source-and-target.html

Reply:
https://github.com/apache/skywalking/blob/cadaab03ae4f01aa983f3f3f157253f17719adc0/apm-sniffer/optional-plugins/lettuce-5.x-plugin/pom.xml#L35
like this may enough

Reply:
@aiyanbo see here 👀

Reply:
@aiyanbo I think, @zhaoyuguang 's point is, one you just need one property, two the following plugin could be removed, because it is already there.

Reply:
@wu-sheng @zhaoyuguang I'm sorry. I lost your comment.